test/P0443_test.cpp

```diff
@@ -30,10 +30,10 @@ namespace {
     void execute(Fn fn) const {
       fn();
     }
-    friend bool operator==(inline_executor, inline_executor) noexcept {
+    [[maybe_unused]] friend bool operator==(inline_executor, inline_executor) noexcept {
```

Comment:
TIL you can put `[[maybe_unused]]` on a function!

Reply:
Me, too! I learned this trick from @tvaneerd's "C++17 in Tony tables": https://github.com/tvaneerd/cpp17_in_TTs/blob/master/attributes.md.

Reply:
Me three!

Reply:
Me four!

I mean, I must have known for a day, but had you asked an hour ago, I would have said I have no idea.